amplify/backend/function/expressServer/src/models/holdings.model.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose')
+const {Schema} = mongoose
+
+const HoldingsSchema = new mongoose.Schema(
```

Comment:
Would an 'average price' field be worthwhile? Rather than calculating a users average cost by going through each and every historical transaction? Maybe not useful for now but for certain metrics like 'person with biggest percent gain' or biggest gain. Or if not it would be an easy metric to display to the user that they are down on 'amzn' overall for example. Helping them play the game better. Just something to think about